### ISSUE TITLE
Only show to users non restricted attributes

### ISSUE THREAD

**Issue by tinova** [OpenNebula/one#6271]
On #6129 a regular user (non admin) could not attach a disk to a VM template.

The cause of this issue is that when you get the disks that could be attached:
![image](https://github.com/OpenNebula/one/assets/134921994/a74e2d99-3d2f-4599-8911-d6791192b1f6)
FSunstonte gets all the info from the API, and when you try to save your template, this includes a field DISK with all the info fields that returns the API. After that, if some of this fields couldn't be change by a regular user (because are in oned.conf) the creation of the VM template will fail.

Also, after correct this error sending to the creation of the VM template the field DISK with only two fields inside (IMAGE and IMAGE_UNAME) there is another problem. The second step (Advanced options) have fields that couldn't be change by a regular user:
![image](https://github.com/OpenNebula/one/assets/134921994/7e9c6379-4330-4330-ad7a-84127f573750)
For example, oned.conf have this line:
`VM_RESTRICTED_ATTR = "DISK/TOTAL_BYTES_SEC"`

And in the form we can set the field by the user: 
![image](https://github.com/OpenNebula/one/assets/134921994/7883ea83-246d-404d-ad2d-6678bc3784fd)

So if the user set this field, the creation will fail.

To summarize:

- In the form of a regular user, the fields that are not allowed to change in oned.conf will have not appear on it. This has to be dinamically because config is in oned.conf
- For an admin user, that not applies, because he could change everything.
- Also, there are other functionalities of the app that needs to have this functionality.

**Comment by dcarracedo** [comment]
This issue was fixed in the development of https://github.com/OpenNebula/one/issues/6154
